README.md

```diff
@@ -46,6 +46,28 @@ git add .
 git commit -m "Commit description"
 git push origin <your-branch>
 ```
+
```

Comment:
- [ ] [Optional] You have some typos in your README, please consider fixing them so your documentation looks more professional